stdlib/universe/hourSelection_test.go

```diff
@@ -0,0 +1,424 @@
+package universe_test
+
+import (
+	"testing"
+	"fmt"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/stdlib/universe"
+)
+
+func TestHourSelection_Process(t *testing.T) {
+	testCases := []struct {
+		name    string
+		spec    *universe.HourSelectionProcedureSpec
+		data    []flux.Table
+		want    []*executetest.Table
+		wantErr error
+	}{
+		{
+			name: "no group key",
+			spec: &universe.HourSelectionProcedureSpec{Start: 17, Stop: 19},
+			data: []flux.Table{
+				&executetest.Table{
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "tag0", Type: flux.TString},
+						{Label: "tag1", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{execute.Time(60000000000001), "a", "b"},
```

Comment:
I really like the format `3600000000000 * 2` because it  is  clearer and makes me understand the test cases. Could you change these tests to use it?

I also think that you can use `time.Hour` instead of `360....`. That is a duration, but it should be easy to change it to execute.Time.

Otherwise you can declare a `const hour int64 = 36000000...` at the top level to increase readability.